packages/app/tests/e2e/features/artifacts/artifactsSet1.feature

```diff
@@ -4,41 +4,32 @@ Feature: Main Page
   Background:
     Given I am on main page
 
-  @id253 @featureEnv @testnet
-  Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
-    Given I click by text "Tools"
-    Given Element with "text" "<Sub-Section>" should be "visible"
-    When Element with "text" "<Sub-Section>" should be "clickable"
-    Then Element with "text" "<Sub-Section>" should have "<url>" value
-
-    Examples:
-      | Sub-Section                 | url                                       |
-      | Smart Contract Verification | /contracts/verify                         |
-      #| Portal                      | https://goerli.staging-portal.zksync.dev/ |
 
-  @id253 @featureEnv @mainnet
+  @id253:I @productionEnv @goerli
   Scenario Outline: Check the element "<Sub-Section>" in Tools section is available, clickable and have correct href
+    Given I go to page "/?network=goerli"
     Given I click by text "Tools"
     Given Element with "text" "<Sub-Section>" should be "visible"
     When Element with "text" "<Sub-Section>" should be "clickable"
     Then Element with "text" "<Sub-Section>" should have "<url>" value
 
     Examples:
-      | Sub-Section                 | url                                       |
-      | Smart Contract Verification | /contracts/verify                         |
-      #| Portal                      | https://staging-portal.zksync.dev/        |
+      | Sub-Section                 | url                                              |
+      | Smart Contract Verification | /contracts/verify                                |
+      | Bridge                      | https://portal.zksync.io/bridge/?network=goerli |
 
-  @id253:I @productionEnv @testnet
+  @id253:I @productionEnv @sepolia
```

Comment:
Split them because both of them are currently present on BE and they have different redirection URLs (href attribute).
I will change the naming for more clear and standardized one.